packages/cli/src/commands/test.js

```diff
@@ -48,6 +48,18 @@ export const builder = (yargs) => {
       type: 'boolean',
       default: false,
     })
+    .option('clearCache', {
```

Comment:
Think we don't need this option anymore :)

Reply:
oh woops! will remove

Reply:
Also think you need to add `db-push` to line 80, so we don't forward it to jest! 

Thanks again :)